dataeng/jobs/analytics/EventExportIncremental.groovy

```diff
@@ -37,9 +37,7 @@ class EventExportIncremental {
                 }
                 publishers common_publishers(allVars)
                 steps {
-                    shell(dslFactory.readFileFromWorkspace('dataeng/resources/opsgenie-enable-heartbeat.sh'))
                     shell(dslFactory.readFileFromWorkspace('dataeng/resources/event-export-incremental.sh'))
-                    shell(dslFactory.readFileFromWorkspace('dataeng/resources/opsgenie-disable-heartbeat.sh'))
```

Comment:
Was this intentionally left in?

Reply:
I had added those bookend build steps to use a Opsgenie heartbeat - but no such heartbeat was provisioned on the Opsgenie side, nor did the extra vars exist in analytics-secure. So I removed the bookends for now.

Were you referring to something else which I "intentionally left in"?